adapter/internal/discovery/xds/server.go

```diff
@@ -54,27 +56,21 @@ var (
 	enforcerSubscriptionPolicyCache    wso2_cache.SnapshotCache
 	enforcerApplicationKeyMappingCache wso2_cache.SnapshotCache
 
-	// API Name:Version -> MgwSwagger struct map
-	apiMgwSwaggerMap map[string]mgw.MgwSwagger
-	// API Name:Version -> Envoy Label Array map
-	openAPIEnvoyMap map[string][]string
-	// API Name:Version -> Envoy Routes map
-	openAPIRoutesMap map[string][]*routev3.Route
-	// API Name:Version -> Envoy Clusters map
-	openAPIClustersMap map[string][]*clusterv3.Cluster
-	// API Name:Version -> Envoy Endpoints map
-	openAPIEndpointsMap map[string][]*corev3.Address
-	// Envoy Label -> XDS version map
-	envoyUpdateVersionMap map[string]int64
-	// Envoy Label -> Listener Configuration map
-	envoyListenerConfigMap map[string]*listenerv3.Listener
-	// Envoy Label -> Routes Configuration map
-	envoyRouteConfigMap map[string]*routev3.RouteConfiguration
-
-	// Enforcer API XDS resource version map
-	enforcerConfigMap map[string][]types.Resource
-
-	openAPIEnforcerApisMap           map[string]types.Resource
+	// API Name:Version as map key
```

Comment:
shall we change the comment such that Vhost:API_Name:Version

Reply:
Updating the comment to Vhost:APIName:Version. Removed the underscore because it wasn't clearly visible in my IDE.